drizztsaga/drizztsaga.tp2

```diff
@@ -190,37 +190,14 @@ COPY ~DRIZZTSAGA\ARE-DATA\BPWM~ ~override~
 
 
 ///////////////////////////////////////////////////////////////////
-//// XP cap patching - now an optional component //////////////////
+//// DEPRECATED: XP cap patching - now an optional component //////
 ///////////////////////////////////////////////////////////////////
 
 BEGIN @1507  // Raise the XP cap
 DESIGNATED 2
-//REQUIRE_COMPONENT "drizztsaga.tp2" "0"  @1503
+DEPRECATED @1515
```

Comment:
So the plan it to remove this component entirely after next release, after the soonest one?

Reply:
Removing components entirely is the easiest way but will break certain scenarios like unattended installation scripts, uninstallation/reinstallation when the mod is already installed etc. Reserving the DESIGNATED number will also safeguard from unintended changes aka "why the second component has the number 3?" etc.